### PR TITLE
updating documents

### PR DESCRIPTION
**Pull request by BengtRydberg**
# Documents
removing DDS, as this doc is replaced by Odin/SMR Product Validation and Evolution Report (we had both on the webpage earlier)

adding IODD  (input/output data document)

and updating five other docs.

ESA told us to do this. 

# Changes to the build pipeline
Npm pulls "chromium" and "chromedriver". Configuration makes sure that the versions matches. Tests runs the binaries pulled from npm.

### CODE REVIEW

**Review by skymandr — APPROVED** [review]
Looks good. There were also comments on PVER, but Donal is working on that and you and/or I probably need to attend to some details as well.

File: docker-compose.systemtest.yml [hunk 1]
```diff
@@ -4,7 +4,6 @@ services:
   postgresql:
     image: odinsmr/odin_db:latest
     ports: ["5432"]
-    restart: unless-stopped
```
Comment (local-minimum) [inline docker-compose.systemtest.yml:7]: why are these removed? Isn't that super annoying if I run the system-tests locally?

File: package.json [hunk 2]
```diff
@@ -30,6 +30,7 @@
   "devDependencies": {
     "babel-loader": "^8.1.0",
     "chromedriver": "^83.0.1",
+    "chromium": "^3.0.2",
```
Comment (local-minimum) [inline package.json:33]: are we still using puppeteer then when we have both chromium and chromedriver dependencies?

**Review by local-minimum — CHANGES_REQUESTED** [review]

File: docker-compose.systemtest.yml (hunk 1 above)
Comment (joakimmoller) [reply docker-compose.systemtest.yml:7]: If a container stops while testing there's probably something wrong.

File: package.json (hunk 2 above)
Comment (joakimmoller) [reply package.json:33]: Yes puppeteer is still used in the test started by "npm test"

Comment (local-minimum) [reply package.json:33]: but puppeteers role is to get chromium and chromedriver versions that match and work, why do we both do that and hard code a version that happens to work right now? Also how are we to maintain these hard-coded versions?

File: docker-compose.systemtest.yml (hunk 1 above)
Comment (local-minimum) [reply docker-compose.systemtest.yml:7]: :facepalm: I was thinking `restart: always` was the default. I was confused.

File: package.json (hunk 2 above)
Comment (joakimmoller) [reply package.json:33]: OK, removed puppeteer, karma is using the versions we selected. Now we have downloading scripts in package-json, the version of those can be maintained by npm. We have to maintain and match the versions of chromebrowser and chromedriver ourselves (set in .npmrc)

Comment (local-minimum) [reply package.json:33]: alright I'm fine with that now that pupeteer is gone.

**Review by local-minimum — APPROVED** [review]
